perses/app/relative_setup.py

```diff
@@ -922,6 +922,62 @@ def _handle_charge_changes(self, topology_proposal, new_positions):
             # modify the topology proposal
             modify_atom_classes(new_water_indices_to_ionize, topology_proposal)
 
+    def _store_phase_topologies(self, phase: str):
+        """
+        Stores topologies and positions in PDB file for the given the phase, for both the initial (old) and final (new)
+        states. Stores both solvent and solute whenever possible (only "solute" in vacuum).
+
+        Generates two PDB files, one for each state (old/new).
```

Comment:
Given that the RCSB is phasing out PDB files in favor of mmCIF files, should we allow the user to control which file format is used?

Reply:
I don't have a strong opinion about this, but I would advocate on just sticking with one to avoid unnecessary complexity, as possible. If we feel we should be trying to push for PDBx/mmcif files, then I'm okay with that. Assuming `openmm.PDBFile` can handle that correctly.

Reply:
Or `openmm.PDBxFile` in that case, I guess.

Reply:
Ok, lets use PDB files for now, but we will want to keep in mind that CIF files are eventually going to replace PDB files, so may want to open an issue about this (for the repo as a whole, not just the issue)

Reply:
Yes, I agree. Created the issue in https://github.com/choderalab/perses/issues/1221